libs/core/src/lib/dialog/dialog-body/dialog-body.component.ts

```diff
@@ -23,6 +23,10 @@ import { DialogRef } from '../utils/dialog-ref.class';
     }
 })
 export class DialogBodyComponent {
+    /** @hidden adds fd-scrollbar class. */
+    @HostBinding('class.fd-scrollbar')
```

Comment:
Instead of adding class to host, please use approach we have in styles https://fundamental-styles.netlify.app/?path=/docs/components-popover--variants#scrollable I mean wrap content with div with scrollbar